tools/codegen/model.py

```diff
@@ -746,7 +755,7 @@ class Argument:
     annotation: Optional[Annotation]
 
     @staticmethod
-    def parse(arg: str) -> 'Argument':
+    def parse(arg: str, cpp_no_default_args: List[str]) -> 'Argument':
```

Comment:
nit: Prefer to do the `name in cpp_no_default_args` test in the call site, and just accept a boolean here

Reply:
But name isn't parsed until inside this function?